public/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="app.css">
+    <title>Turn Based Game</title>
+</head>
+<body>
+
+<div class="game">
+	<div class="start"></div>
+	<div class="battle"></div>
+	<div class="end"></div>
+</div>
+
+<script src="vendor.js"></script>
+<script src="jquery/dist/jquery.min.js"></script>
```

Comment:
jquery.min.js and handlebars.js are part of `vendor.js` so you don't need them in your index.html. They don't exist as their own files in the final build.